index.html

```diff
@@ -199,13 +206,15 @@ <h3>Skills</h3>
       </ul>
     </section>
     <footer>
-      <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
-      <form action="https://formspree.io/f/mnqwwwwq" method="post">
-        <input type="text" name="name" required maxlength="30" placeholder="Full Name">
-        <input type="email" name="email" required placeholder="Email address">
-        <textarea name="message" cols="30" rows="7" maxlength="500" required>Enter text here</textarea>
-        <button type="submit">Get in touch</button>
-      </form>
+      <div class="title-form">
+        <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+        <form action="https://formspree.io/f/mnqwwwwq" method="post">
+          <input type="text" name="name" required maxlength="30" placeholder="Full Name">
+          <input type="email" name="email" required placeholder="Email address">
```

Comment:
- [ ] please it would be better if you add some padding in your inputs in order to match The Figma design 
![image](https://user-images.githubusercontent.com/61790587/171491774-9241c2eb-d91d-48f9-8ebb-849f58cd021b.png)